src/TinyCiv.Shared/Events/Server/LobbyStateServerEvent.cs

```diff
@@ -0,0 +1,3 @@
+namespace TinyCiv.Shared.Events.Server;
+
+public record LobbyStateServerEvent(bool CanGameStart) : ServerEvent;
```

Comment:
@Randwarf @jshimas now ```ListenForLobbyState``` (previously was ```ListenForGameStartReady```) will return bool ```CanGameStart``` to indicate if the game can still be started

Basically: two players join -> ListenForLobbyState callback called -> CanGameStart = true -> one player leaves (can send  ```LeaveLobbyClientEvent``` or just close client window) ->  ListenForLobbyState callback called ->  CanGameStart = false

If player leaves during the game, then we ignore that fact and let the first player finish (win) the game